pkg/broker/registry.go

```diff
@@ -77,7 +86,9 @@ func MapServiceIdToName() map[string]string {
 	out := map[string]string{}
 
 	for _, svc := range brokerRegistry {
-		out[svc.CatalogEntry().ID] = svc.Name
+		if entry, err := svc.CatalogEntry(); err == nil {
```

Comment:
Not to dogpile onto this code, but you should probably handle the error case as well :)

Reply:
Fixed, I refactored to remove the need for this method. It was going to be tossed eventually so better sooner rather than later.